x/emissions/keeper/inference_synthesis/synth_palette_forecast_implied.go

```diff
@@ -75,10 +77,15 @@ func (p SynthPalette) CalcForecastImpliedInferences() (map[Worker]*emissionstype
 					if err != nil {
 						return nil, errorsmod.Wrapf(err, "error calculating network loss per value")
 					}
-					R_ik[j] = StatefulRegret{regret: R_ijk, noPriorRegret: false}
+					R_ik[j] = &StatefulRegret{regret: R_ijk, noPriorRegret: false}
+				}
+
+				if sortedInferersInForecast != nil && len(sortedInferersInForecast) > 0 {
```

Comment:
Maybe this function should simply never be called if there are no inferers... that's what we try to enforce everywhere else

Reply:
which function are you talking about `CalcForecastImpliedInferences` or `CalcWeightsGivenWorkers` ?

Reply:
both actually. if there are no inferers, we shouldn't have even ever accepted the worker payload